### PR TITLE
Solve issue #432

### PR DESCRIPTION
**Pull request by lmm-git**
refs #432

### CODE REVIEW

File: docs/license.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ these elements still apply and are not touched by the software license.
 ::
 
   EvaP - Evaluation Platform
-  Copyright (C) 2011-2014 by Michael Grünewald, Stefan Richter, Johannes Linke and Johannes Wolf
+  Copyright (C) 2011-2015 by Michael Grünewald, Stefan Richter, Johannes Linke and Johannes Wolf
```
Comment (cmfcmf) [inline docs/license.rst]: It looks like this commit shouldn't be in this PR.

Comment (lmm-git) [inline docs/license.rst]: But it should be no problem, or?

Comment (janno42) [inline docs/license.rst]: Don't worry, we'll clean it up. Next time please make sure to not commit existing changes again :)

**Comment by janno42** [comment]
to test whether so. is editor or delegate or contributor is not necessary, because all editors must be contributors. could you please try to refactor the code according to this?

**Comment by lmm-git** [comment]
Do you mean it like this?

File: evap/contributor/views.py [hunk 2]
```diff
@@ -105,13 +105,13 @@ def course_edit(request, course_id):
...
 
-@editor_or_delegate_required
+@delegate_or_contributor_required
 def course_preview(request, course_id):
     user = request.user
     course = get_object_or_404(Course, id=course_id)
 
     # check rights
-    if not (course.is_user_editor_or_delegate(user) and course.state in ['prepared', 'lecturerApproved', 'approved', 'inEvaluation', 'evaluated', 'reviewed']):
+    if not ((course.is_user_editor_or_delegate(user) or course.is_user_contributor(user)) and course.state in ['prepared', 'lecturerApproved', 'approved', 'inEvaluation', 'evaluated', 'reviewed']):
```
Comment (janno42) [inline evap/contributor/views.py]: please remove the editor here as well

**Comment by janno42** [comment]
yes, thanks. i think there is just one more place where you could remove the editor check.

**Comment by janno42** [comment]
replaced by #511

**Pull request closed without merging**